src/diagonal.works/b6/ingest/change.go

```diff
@@ -98,6 +99,27 @@ func (a *AddFeatures) Apply(w MutableWorld) (AppliedChange, error) {
 		}
 	}
 
+	for _, collection := range a.Collections {
+		if len(a.IDsToReplace) > 0 {
+			for i, key := range collection.Keys {
+				if id, ok := key.(b6.Identifiable); ok {
+					if ns, ok := a.IDsToReplace[id.FeatureID().Namespace]; ok {
+						newID := b6.FeatureID{id.FeatureID().Type, ns, id.FeatureID().Value}
```

Comment:
ugh this is weird, and prob not cool - i m losing the original feature type - should i get it from values[i] / as i can assume it s a feature if feature id is it s key ? or fine like this ?

Reply:
I don't think we actually need this anymore. It's a hangover from a time when we optionally allocated new IDs when a feature was added, which was necessary partly because namespaces couldn't be created dynamically by inventing new namespace strings - there were a limited number of pre-defined namespaces. I think it's better you just delete the code here and add a TODO to say the adding collections doesn't support ID replacement, and the whole thing should be deleted.

Reply:
aight